gravitee-apim-console-webui/angular.json

```diff
@@ -166,7 +166,8 @@
           "builder": "@angular-devkit/build-angular:dev-server",
           "options": {
             "proxyConfig": "proxy.conf.mjs",
-            "browserTarget": "apim-console:build"
+            "browserTarget": "apim-console:build",
+            "disableHostCheck": true
```

Comment:
This is because my local environment is configured to work with vhost and I noticed that, since angular migration, it wasn't working anymore. I've checked with @ThibaudAV.